src/CSSBattle/8_ForkingCrazy/ZhangZhikai.html

```diff
@@ -0,0 +1,113 @@
+<div class="botsemi"></div>
+<div class="rect">
+</div>
+<div class="smallsemi">
+</div>
+<div class="rect botsec">
+</div>
+<div class="smallsemi botsec">
+</div>
+<div class="rect botthir">
+</div>
+<div class="smallsemi botthir">
+</div>
+<div class="rect top">
+</div>
+<div class="topsmallsemi">
+</div>
+<div class="topsmallsemi topsec">
+</div>
+<div class="rect top topsec">
+</div>
+<div class="topsmallsemi topthir">
+</div>
+<div class="rect top topthir">
+</div>
+<div class="topsmallsemi topfour">
+</div>
+<div class="rect top topfour">
+</div>
```

Comment:
`<div></div>` can perhaps be on a single line to make your code neater

Reply:
50ed92b92dcb6d7fac9a9b6965067d8e1abde559